sql/files/defaultdata/ocaml/run

```diff
@@ -8,10 +8,12 @@ MEMLIMIT="$1" ; shift
 # Compilation needs to store intermediate files, `/tmp` is not writable
 export TMPDIR="."
 
-# Enable some libraries
-ocamlopt unix.cmxa str.cmxa bigarray.cmxa -o "$DEST" "$@" -ccopt -pipe
-
+# Enable some libraries;
+# unix system calls, regular expressions and string processing & multi arrays
+# See: https://v2.ocaml.org/releases/5.1/api/index.html
 # Consider also setting for faster executables:
 # -match-context-rows 100
 # -unsafe
+ocamlopt unix.cmxa str.cmxa bigarray.cmxa -o "$DEST" "$@" -ccopt -pipe
```

Comment:
Is there a good reason to add specifically these libraries? Why (not) add others as well? I think it's better to go with language defaults unless there's clearly a general reason to add/change something. We can just document some options and local admins can always customize things.

Reply:
This is what SWERC24 uses, so I assumed this is the reasonable default. Also I prefer to have an example of how to add the extra libraries.

But if you really prefer I can remove this and add it as a comment.